server/src/main/scala/za/co/absa/atum/server/api/http/HttpEnv.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.api.http
+
+import za.co.absa.atum.server.api.controller.{CheckpointController, PartitioningController}
+import zio.RIO
+import zio.metrics.connectors.prometheus.PrometheusPublisher
+
+object HttpEnv {
```

Comment:
@benedeki 
I am not sure I follow. Can you elaborate a bit more on the idea?

Reply:
```suggestion
package object http {
```
... and renaming the file to `package.scala`

Reply:
I see. I was actually considering this but then I was thinking there could be clashes as type `F` can be used in other places for different purposes. That's why I decided to keep it as normal object and use it everywhere as `HttpEnv.Env` and `HttpEnv.F` to make is absolutely clear which type I am referring to and to avoid potential clashes.